sql/parser/update.go

```diff
@@ -56,12 +57,18 @@ func (p *Parser) parseSetClause() ([]updateSetPair, error) {
 			}
 		}
 
-		// Scan the identifier for the field name.
 		tok, pos, lit := p.ScanIgnoreWhitespace()
```

Comment:
There is no need to scan for an identifier if we are going to parse a field ref, I think this should be removed

Reply:
Indeed